docs/contracts/db-additions.sql

```
@@ -0,0 +1,340 @@
+    last_log_id     INTEGER,                     -- references logs.id of the triggering row
+    severity        TEXT    NOT NULL,            -- 'info' | 'warn' | 'critical'
+    ack_at          INTEGER,                     -- NULL while active; unix ms when acked
+    ack_by          TEXT,                        -- 'auto' | 'mcp:<user>' | 'timeout' | 'quiet_hours'
+    snooze_until    INTEGER,                     -- NULL or unix ms
+    PRIMARY KEY (rule_id, fingerprint)
+);
+
+-- "All currently active alerts" — partial index over the active set only.
+-- (Source: §6)
+CREATE INDEX IF NOT EXISTS idx_alert_state_active
+    ON alert_state(ack_at) WHERE ack_at IS NULL;
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)